src/v3/actions/message.ts

```diff
@@ -70,19 +92,43 @@ export function createMessageActions(ontime: OntimeV3): { [id: string]: Companio
 			],
 			callback: ({ options }) =>
 				socketSendJson(ActionCommand.Message, {
-					[options.destination as string]: { text: options.value },
+					timer: { text: options.value },
 				}),
 		},
-
+		[ActionId.MessageVisibilityAndText]: {
+			name: 'Toggle/On/Off visibility and text for message',
+			description:
+				'Combinde action for setting the text and visibility. "Toggle" will swap out the message if another is showen and "Off" will only turn off if this message is visible',
```

Comment:
Maybe we can reduce the text here
"Toggle" will swap out the message if another is showen and "Off", we can just say that it will replace the current message

"Off" will only turn off if this message is visible: maybe we dont need to explain. off will turn it off, if it was already off, there is nothing to do 

```suggestion
			description:
				'Combined action for setting the text and visibility. "Toggle" will replace the current message. "Off" will disable the message visibility',
```